src/qml/qgismobileapp.qml

```diff
@@ -3931,4 +3932,38 @@ ApplicationWindow {
       pluginPermissionDialog.open();
     }
   }
+
+  function setupGuides() {
+    const startupGuide = settings.valueBool("/QField/showMapButtonsGuide", true);
+    if (startupGuide) {
+      runStartupTour();
+    }
+    settings.setValue("/QField/showMapButtonsGuide", false);
+  }
+
+  function runStartupTour() {
+    startupTour.open();
+  }
+
+  QFieldGuide {
+    id: startupTour
+    baseRoot: mainWindow
+    steps: [{
+        "title": qsTr("Zoom Toolbar"),
+        "description": qsTr("Here you can zoom in/out easily through these buttons."),
+        "target": () => [zoomToolbar]
+      }, {
+        "title": qsTr("Menu"),
+        "description": qsTr("You can open dashboard here, see map legends or start digitizing."),
```

Comment:
I think I'd start with this one, that's essentially the primary button.

```suggestion
        "description": qsTr("You can open dashboard here to interact with the project's legends and map theme, or start digitizing by activating the editing mode. Long-pressing the button also gives you immediate access to the main menu."),
```

Reply:
done 👍